models/User.js

```diff
@@ -51,6 +51,12 @@ const userSchema = new Schema({
     lowercase: true,
     unique: true,
   },
+  personalEmail: {
+    type: String,
+    required: true,
```

Comment:
Question: Did anyone ever specify that we were making personal emails require to register with shpeuf?

Reply:
Mariana